packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -138,8 +138,14 @@ export const messages = {
   force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
   force_function_create_text: 'SFDX: Create Function',
   force_function_start_text: 'SFDX: Start Function',
+  force_function_start_warning_no_toml:
+    'No function.toml found. Create function.toml or intialize a new function folder using SFDX: Create Function.',
+  force_function_start_warning_not_in_function_folder:
```

Comment:
This happens when you run "SFDX: Start Function" in command palette, but haven't open a file function folder. See the comments in https://github.com/forcedotcom/salesforcedx-vscode/pull/2498/files#r488314698

Reply:
Drop the "please" - we usually don't use please unless we're asking customer to do something VERY unpleasant. So: Open a function file to run SFDX: Start Function.